code-challenges.test.js

```diff
@@ -12,15 +12,66 @@
 
 // --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
 
-// a) Create a test with expect statements for each of the variables provided.
+// // a) Create a test with expect statements for each of the variables provided.
 
 const fibLength1 = 6
 // Expected output: [1, 1, 2, 3, 5, 8]
 
 const fibLength2 = 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
-// b) Create the function that makes the test pass.
+describe("myNumber", () => {
+  it("returns an array that length containing the numbers of the Fibonacci sequence.", () => {
+    expect(myNumber()).toEqual([1, 1, 2, 3, 5, 8])
+  })
+})
+
+
+// // ReferenceError: myNumber is not defined
+
+
+// // Pseudocode : Create a function named myNumber
+// // Input Takes in a number (greater than 2)
+//    For loop to iterate through the 'array" and .push to add to the end of the array, at this point im confused on why its not working.
```

Comment:
One of the things we have to consider is where our return is , as well as what we are asking to return.